packages/fab/package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "@material/fab-react",
+  "version": "0.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
```

Comment:
I don't think you need a scripts section, or most of this package.json, now that we have lerna setup

Reply:
What should these components be called? Currently we have `@material/top-app-bar-react` and `@material/fab-react`.

Reply:
@material/fab-react